cloudinit/config/cc_scripts_vendor.py

```diff
@@ -3,12 +3,17 @@
 # Author: Ben Howard <ben.howard@canonical.com>
 #
 # This file is part of cloud-init. See LICENSE file for license information.
+"""Scripts Vendor: Run vendor scripts"""
 
-"""
-Scripts Vendor
---------------
-**Summary:** run vendor scripts
+import os
+
+from cloudinit import subp, util
+from cloudinit.config.schema import MetaSchema, get_meta_doc
+from cloudinit.distros import ALL_DISTROS
+from cloudinit.settings import PER_INSTANCE
 
+frequency = PER_INSTANCE
+MODULE_DESCRIPTION = """\
 Any scripts in the ``scripts/vendor`` directory in the datasource will be run
 when a new instance is first booted. Scripts will be run in alphabetical order.
```

Comment:
I think we should update docs to somehow mention the run order of script config modules is [vendor, per-once, per-boot, per-instance, user].

Reply:
If one cares about ordering, they would need to check /etc/cloud/cloud.cfg, and I think we need to find a way to document that separately. I don't disagree that it should be documented, but I'm not sure the module documentation is the best place for it.